README.md

```diff
@@ -137,6 +137,12 @@ To save slug space, the `trace-agent` and `process-agent` optional binaries are
 
 To reduce your slug size, make sure that `DD_APM_ENABLED` is set to `false`, if you are not using APM features; and `DD_PROCESS_AGENT` is not set to `true`, if you are not using process monitoring.
 
+## Python and Agent versions
+
+Prior to version `6.14` the Datadog v6 agent shipped with Python2 embedded. Starting with `6.14`, and in preparation for Python2 End Of Life, announced for January 2020, the Datadog v6 agent ships with both Python2 and Python3, to give customers enough time to migrate their custom checks to Python3. The Heroku buildpack will only keep one of the versions. Set `DD_PYTHON_VERSION` to `2` or `3` to select the Python version you want the agent to keep. If not set, the buildpack will keep Python2. If you are using custom checks that only work with Python2, we recommend to migrate them to Python3 before its EOL.
```

Comment:
Minor thing, but you use `Python versions 2 and 3` above, and `Python2` and `Python3` here. Don't care which you choose to describe, but use one and be consistent.